src/main/java/cinema/dao/MovieDaoImpl.java

```diff
@@ -0,0 +1,46 @@
+package cinema.dao;
+
+import cinema.exception.DataBaseException;
+import cinema.lib.Dao;
+import cinema.model.movie.Movie;
+import cinema.util.HibernateUtil;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+@Dao
+public class MovieDaoImpl implements MovieDao {
+    @Override
+    public Movie add(Movie movie) throws DataBaseException {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            Long itemId = (Long) session.save(movie);
+            transaction.commit();
+            movie.setId(itemId);
```

Comment:
```suggestion
            session.save(movie);
            transaction.commit();
```
look [closely](https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_01.html)